packages/stargate/src/aminotypes.ts

```diff
@@ -3,35 +3,6 @@ import { AminoMsg } from "@cosmjs/amino";
 import { EncodeObject } from "@cosmjs/proto-signing";
 
 import { AminoConverter, AminoConverters } from "./aminoconverters";
-import {
-  createAuthzAminoConverters,
-  createBankAminoConverters,
-  createDistributionAminoConverters,
-  createFreegrantAminoConverters,
-  createGovAminoConverters,
-  createIbcAminoConverters,
-  createStakingAminoConverters,
-} from "./modules";
-
-function createDefaultTypes(prefix: string): AminoConverters {
-  return {
-    ...createAuthzAminoConverters(),
-    ...createBankAminoConverters(),
-    ...createDistributionAminoConverters(),
-    ...createGovAminoConverters(),
-    ...createStakingAminoConverters(prefix),
-    ...createIbcAminoConverters(),
-    ...createFreegrantAminoConverters(),
-  };
-}
```

Comment:
After removing this here, you can probably move back the types `AminoConverters` and `AminoConverter` to this file. I moved them to `packages/stargate/src/aminoconverters.ts` because we had cyclic imports.